recipes/fluiddyn/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "fluiddyn" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 488dcc8cd1660fd6eed939defeb21250114b5de9c94cc1d979136ea439789b33
+
+build:
+  number: 0
+  entry_points:
+    - fluidinfo = fluiddyn.util.info:main
+    - fluidnbstripout = fluiddoc.fluidnbstripout:main
+    - fluiddocset = fluiddoc.fluiddocset:main
+    - fluiddump = fluiddyn.io.dump:main
+    - fluidmat2py = fluiddyn.util.matlab2py:main
+    - fluidcluster-help = fluiddyn.clusters:print_help_scheduler
+  script: "{{ PYTHON }} -m pip install .[full] --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy
+    - psutil  >=5.2.1
+    - six
+    - h5py
+    - nbstripout
+
+test:
+  imports:
+    - fluiddoc
+    - fluiddyn
+    - fluiddyn.calcul
+    - fluiddyn.calcul.test
+    - fluiddyn.clusters
+    - fluiddyn.clusters.test
+    - fluiddyn.io
+    - fluiddyn.io.test
+    - fluiddyn.output
+    - fluiddyn.output.test
+    - fluiddyn.util
+    - fluiddyn.util.matlab2py
+    - fluiddyn.util.matlab2py.test
+    - fluiddyn.util.test
+  commands:
+    - fluidinfo --help
+    - fluidnbstripout --help
+    - fluiddocset --help
+    - fluiddump --help
+    - fluidmat2py --help
+    - fluidcluster-help --help
+
+about:
+  home: https://bitbucket.org/fluiddyn/fluiddyn
+  license: CeCILL-B
+  license_family: BSD
```

Comment:
This licence doesn't look like a BSD licence to me.
```suggestion
```

Reply:
This licence

> This licence doesn't look like a BSD licence to me.
> Suggested change
>   license_family: BSD

Are you sure?
The Cecill-B license has the caracteristics of BSD-like licenses : http://www.cecill.info/licences.en.html

Reply:
Sorry I worded that comment poorly.

It might be compatible but I wouldn't think of it as being part of the [BSD family of licences](https://en.wikipedia.org/wiki/BSD_licenses) as it contains notable differences such as:

> 10.1 In the event of a breach by the Licensee of its obligations hereunder, the Licensor may automatically terminate this Agreement thirty (30) days after notice has been sent to the Licensee and has remained ineffective.

and

> 13.2 Failing an amicable solution within two (2) months as from their occurrence, and unless emergency proceedings are necessary, the disagreements or disputes shall be referred to the Paris Courts having jurisdiction, by the more diligent Party.

It's not a problem and the licence looks perfectly reasonable, I'd just remove the `license_family: BSD` line to avoid confusion.